.gitlab/trigger_release/trigger_release.yml

```diff
@@ -55,3 +55,32 @@ trigger_manual_prod_release:
     - if: $CI_COMMIT_TAG =~ /^[0-9]+\.[0-9]+\.[0-9]+-v[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+){0,1}$/
       when: never
     - !reference [.on_deploy_stable_or_beta_manual_auto_on_stable]
+
+include:
+  - https://gitlab-templates.ddbuild.io/slack-notifier/v3-sdm/template.yml
+
+.setup_macos_github_app_for_runner_bump:
+  # GitHub App rate-limits are per-app. Since we are rarely calling the job, we are only using the instance 2
+  - |
+    GITHUB_KEY_B64=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $MACOS_GITHUB_APP_2 key_b64) || exit $?; export GITHUB_KEY_B64
+    GITHUB_APP_ID=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $MACOS_GITHUB_APP_2 app_id) || exit $?; export GITHUB_APP_ID
+    GITHUB_INSTALLATION_ID=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $MACOS_GITHUB_APP_2 installation_id) || exit $?; export GITHUB_INSTALLATION_ID
+    echo "Using GitHub App instance 2"
+
+generate_windows_gitlab_runner_bump_pr:
+  stage: trigger_release
+  extends: .slack-notifier-base
+  needs: ["trigger_auto_staging_release"]
+  tags: ["arch:amd64"]
+  rules:
+    - if: $DDR == "true"
+      when: never
+    - if: $CI_COMMIT_TAG =~ /^[0-9]+\.[0-9]+\.[0-9]+-v[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+){0,1}$/
+      when: never
+    - if: $CI_COMMIT_TAG =~ /^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?$/
+
+  script:
+    - !reference [.setup_macos_github_app_for_runner_bump]
```

Comment:
It might be nice to add a comment that explains the choice of this github app.

Reply:
I meant more specifically why we're using one with "macos" in the name; I know that we have at least one other app (Agent Platform Auto PR) and it is surprising to see such a reference when we're doing something that has nothing to do with macos.

Reply:
It is actually `Agent Platform Auto PR` that is the macOS one but yea sure I'll edit the comment 👍